events/artsy-learns-to-code-v2.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Artsy Learns to Code v2
+description: Second iteration on ALTC
+---
+
+See the original ALTC write-up [here](./artsy-learns-to-code.md) for the previous iteration's abstract and
+structure.
+
+## Abstract
+
+Let's kick off another session of Artsy Learns to Code!
+
+## Motivations for ALTC v2
+
+- It's social! Lets Artsy people interact across their normal silos
+- Build empathy across Artsy
+- It's fun! Boosts morale
+- Empowers non-technical people
+- Gets engineers used to teaching regularly
+
+## Features
+
+- Learners can opt to specify different areas of engineering/CS they'd like to learn about. We'll do our best to
+  accomodate these requests, but won't guarantee that we can match mentors' expertises with learners' requests.
+- If students don't have a specific idea for what they'd like to learn (which is totally ok!), we have a default
+  suggested teaching plan: p5.js aka Processing in JS.
+
+## Structure
+
+Similar to the previous ALTC, we'll have trios: one mentor and two learners, who will be partners throughout _Artsy
+Learns to Code_.
```

Comment:
Last time we immediately had to expand to 4 learners per mentor. Should we just build that in this time around?

Reply:
Hmm, maybe let's gauge interest and see? I really like the idea of smaller more intimate groups: fewer learners get left behind and the barrier for questions is lower.